### ISSUE TITLE
Feature: Setup basic layout of WCAG Page

### ISSUE THREAD

**Issue by michaeldev6** [michaeldev6/dev-a11y#3]
### Setup simple layout for the WCAG page
- an area that shows the full list
- list should be pulled from from the `/shared/constants/wcag/guidelines/wcag-guidelines.ts`
- page should have a section to filter and sort the list. Possible on the right or left. Maybe on the left so it is hit first when tabbing through.
  - Order by WCAG Id
  - Group by Levels
  - Group by tags (`/shared/enums/wcag-tags.ts`)
  - Filter out by levels or tags
- Each items should be a link to the details of the WCAG item (`/wcag/{{wcagId}}`)
